specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimskus.json

```diff
@@ -0,0 +1,364 @@
+          "items": {
+            "$ref": "#/definitions/ApiManagementSkuLocationInfo"
+          },
+          "description": "A list of locations and availability zones in those locations where the SKU is available."
+        },
+        "apiVersions": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "The api versions that support this SKU."
+        },
+        "costs": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ApiManagementSkuCosts"
+          },
+          "description": "Metadata for retrieving price info."
+        },
+        "capabilities": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ApiManagementSkuCapabilities"
+          },
+          "description": "A name value pair to describe the capability."
+        },
+        "restrictions": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ApiManagementSkuRestrictions"
+          },
+          "description": "The restrictions because of which SKU cannot be used. This is empty if there are no restrictions."
+        }
+      },
+      "description": "Describes an available ApiManagement SKU."
+    },
+    "ApiManagementSkuLocationInfo": {
+      "properties": {
+        "location": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Location of the SKU"
+        },
+        "zones": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of availability zones where the SKU is supported."
+        },
+        "zoneDetails": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiManagementSkuZoneDetails"
+          },
+          "description": "Details of capabilities available to a SKU in specific zones."
+        }
+      }
+    },
+    "ApiManagementSkuRestrictionInfo": {
+      "properties": {
+        "locations": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Locations where the SKU is restricted"
+        },
+        "zones": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of availability zones where the SKU is restricted."
+        }
+      }
+    },
+    "ApiManagementSkusResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiManagementSku"
+          },
+          "description": "The list of skus available for the subscription."
+        },
+        "nextLink": {
```

Comment:
>nextLink [](start = 9, length = 8)

add readOnly: true to this property as well

Reply:
Do we add read-only  to `next link` ?

Reply:
Yes please, readOnly: true is set to all the properties that your RP sets and are not allowed to be updated by an end-user